pages/board_recruiting.vue

```diff
@@ -1,13 +1,11 @@
 <template>
```

Comment:
Can we just delete this file? It will be in the version control if we want to restore it.

Reply:
I left this page in as there will be recruitment in the future. The content was updated to reflect that there isn't any open positions.

I didn't think that removing and reinstating the page is the best practice as we might end up with dead links (I'm sure we have shared the recruiting page in slack and emails).

I can still remove it if that's your preference, but I don't see any harm in keeping it active?

Reply:
Sure, that makes sense I suppose. You're right about not wanting to break any links.